src/test/resources/org/broadinstitute/hellbender/tools/haplotypecaller/testPloidyRegions.bed

```diff
@@ -0,0 +1,3 @@
+20	10000000	10010000	2
+20	10020000	10030000	4
```

Comment:
can you put into the test file a demonstration of the rule "overlapping ployidy defaults to the highest option?"

Reply:
Added a method for getting current ploidy to trim down that part of the code. Still need to add the test